Set vscroll by pk

PySimpleSQL opened the pull request.

ssweber commented:
I can cleanup, sync my development and just send over the vscroll part. I hit the get_min_pk issue, and got confused if I was creating the problem or something else.

PySimpleSQL commented:
Its all good, i just moved it over by hand (still getting my head around
Github on some of these issues, having come from Subversion).  I just added
the same vscroll position behavior for listboxes too

On Sun, Feb 12, 2023 at 12:53 PM ssweber ***@***.***> wrote:

> I can cleanup, sync my development and just send over the vscroll part. I
> hit the get_min_pk issue, and got confused if I was creating the problem or
> something else.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PySimpleSQL/pysimplesql/pull/62#issuecomment-1427092624>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQX2REWAXWR6XWIMVEHTBODWXEPS3ANCNFSM6AAAAAAUZN3RIQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
